content/events/events-content.md

```diff
@@ -2,12 +2,33 @@
 
 <!--The text at the top of this page is pulled from `events-snippet.md` in this folder-->
 
-ECHO includes compliance data on air, water, and hazardous waste laws for all facilities permitted to emit, store, and use hazardous materials along with data on inspections and enforcement actions taken by EPA and state agencies. It is the first database to publicly share federal environmental enforcement data. However, the website itself can be hard for non-specialists to use and understand. To increase ECHO’s public accessibility, the EEW team is developing a series of Jupyter Notebooks – collaborative, open-source data analysis tools – so people can use this information from ECHO to make customizable reports on pollution from facilities in areas of interest.
+![Democratizing Environmental Data Panel Discussion and Report Card Release Event Poster](./report_release_event.png)
 
-These reports can include information about EPA inspections, whether facilities are complying with federal regulations, and if EPA has responded to violations with enforcement actions. We’ve already written the code to generate reports based on the data—during the workshops, participants simply run the notebook to generate the data for their area of interest. No coding or data science experience is needed!
+**Want to attend? Register up [here](https://us02web.zoom.us/webinar/register/WN_dAn8xJF9Tqm1aiC1A8Ysig)**
```

Comment:
This line might look better centered (add `<center>` and `</center>` around it)

Reply:
I tried centering it but it doesn't look centered when I preview it and instead of adding the link to the text it just prints out the code. How should I fix it?